### PR TITLE
Cleanup GFlash and PhysicsLists sub-libraries

### PR DESCRIPTION
**Pull request by civanch**
Obsolete unused classes have been removed from GFlash subdirectory.

Few classes were cleaned up. Unnecessary Geant4 disclaimers are removed.

### CODE REVIEW

**Comment by civanch** [comment]
please test

**Comment by civanch** [comment]
+1
this PR is trivial

File: SimG4Core/PhysicsLists/interface/UrbanMscModel93.h [hunk 1]
```diff
@@ -1,52 +1,29 @@
-//
```
Comment (davidlange6) [inline SimG4Core/PhysicsLists/interface/UrbanMscModel93.h]: @civanch - in terms of removing the g4 license. what is your estimate of code overlap with the original UrbanMscMode93? It seems quite high to me.

**Comment by civanch** [comment]
The model itself was not fully stable and was changed a bit from release to release. In Geant4 used in 8_0_X Urban93 model does not exist any more. It was one of the mail reasons to copy the model to CMSSW. By the way, I have full agreement with Laszlo Urban about authorship statement within CMSSW version of the code.

**Comment by davidlange6** [comment]
@civanch - maybe you can forward me some more information and add a note in the code? [I'm not sure the authors permission is sufficient in this case, but at least we will remember the process we went through if asked in a few years]

**Comment by civanch** [comment]
@davidlange6 , I have prepared an extended comment to the header of the class. If you will be satisfied by it will be committed. Please, have a look below:

//
// GEANT4 Class header file
//
//
// File name: UrbanMscModel93
//
// Original author:    Laszlo Urban, 
//
//    V.Ivanchenko have copied from G4UrbanMscModel93 class
//                 of Geant4 global tag geant4-09-06-ref-07 
//                 and have adopted to CMSSW
//
// Creation date: 21.08.2013 
//
// Class Description:
//
// Implementation of the model of multiple scattering based on
// H.W.Lewis Phys Rev 78 (1950) 526 
// L.Urban CERN-OPEN-2006-077, Dec. 2006
// V.N.Ivanchenko et al., J.Phys: Conf. Ser. 219 (2010) 032045

// -------------------------------------------------------------------
// In its present form the model can be  used for simulation 
//   of the e-/e+, muon and charged hadron multiple scattering
// 
// This code was copied from Geant4 at the moment when it was removed 
// from Geant4 completly (together with G4UrbanMscModel91, 95, 96).
// Since that time Geant4 supports the unique class G4UrbanMscModel.
// It was shown in Geant4 internal validations that this last class
// provides more accurate simulation for various thin target tests.
// However, it is not guranteed stable results for calorimeters.
// CMS private version of the Urban model has the same performance as 
// the model used for run1 but includes several technical fixed introduced
// after run1 (mainly to avoid numerical problems for very small steps).

**Comment by civanch** [comment]
please test

**Comment by civanch** [comment]
please test

**Comment by civanch** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**